test/fixtures/profiles/old-examples/profile/controls/example-namespace-clash.rb

```diff
@@ -0,0 +1,8 @@
+title "Namespace clashing control"
+
+control 'my_control' do
+only_if { input('my_control', value: 'false') == 'false' }
+  describe file("/tmp") do
+    it { should be_directory }
+  end
+end
```

Comment:
I'd suggest avoid making any changes to the old-examples fixtures - those are a relic from when user examples were used as test fixtures. This might be better suited under `fixtures/profiles/waivers` or `fixtures/profiles/inputs` . If you need to make a new profile, use `inspec init profile`. 

Placing it here impacts a lot of tightly coupled things like test counts, which could be avoided.

Reply:
Thanks for the feedback on this. I'm still building up my internal knowledge on how to test things here so that type of input is very much appreciated. I'll move this all over to `fixtures/profiles/waivers` to keep in flow 👍